docusaurus/docs/reference/tunnelers/linux/index.md

```diff
@@ -223,23 +241,27 @@ wget -q "https://github.com/openziti/ziti-tunnel-sdk-c/releases/latest/download/
 
 ### Enroll Before You Run
 
-You will need the token file or its contents to enroll. Enrollment is the act of exchanging the token for an identity that is to be permanently installed in the filesystem.
+You will need the token file or its contents to enroll. 
+
+```text
+ziti-edge-tunnel enroll --jwt ./in-file.jwt --identity ./out-file.json
+```
 
-[Learn more about enrolling](/docs/learn/core-concepts/identities/enrolling).
+[Learn more about enrolling](/learn/core-concepts/identities/20-enrolling.md).
 
 ### Run the Manually Installed Binary
 
 ```text
-ziti-edge-tunnel run \
-  --identity-dir /opt/openziti/etc/identities
+ziti-edge-tunnel run --identity-dir /opt/openziti/etc/identities
 ```
 
-[Learn more about tunneler options and modes](./linux-tunnel-options).
+[Learn more about tunneler options and modes](./linux-tunnel-options.md).
 
 ## Run with Docker
 
-Please reference [the article about running the Linux tunneler in a container](./container/readme.mdx) for guidance and examples!
+Please reference [the article about running the Linux tunneler in a container](./container/readme.mdx) for guidance and
```

Comment:
pleasantries and exclams are also things to look for / avoid. reference doc should just be factual doc